data_acquisiton/twitter-credentials.json

```diff
@@ -1,6 +0,0 @@
-{
-    "consumer_key": "Y5bv4r7Ob6sYOVA6EQ9xpHMxX",
-    "consumer_secret": "81x3J1RMwop6TC8NgMWAxAqLswDoUoACgkJ40AKyTy8GumqCuj",
-    "access_token": "4634465892-n2vEUdUZsuoHweFwJbJ6075ZFc9I7MUd3tyDZCu",
-    "access_token_secret": "Kxaz2EKBJQAOu62y6nPrufXCtS9lDCiAHjGFpoOaUIm5j"
-}
```

Comment:
I will be a good idea to request a change to this information. Any sensitive information that was a part of the previous repository should be changed.